src/components/EmbeddedWalletLayout/styles.scss

```
@@ -0,0 +1,134 @@
+.EmbeddedWalletLayout {
+  position: fixed;
+  inset: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: var(--sds-clr-gray-02);
+  box-sizing: border-box;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+.EmbeddedWalletHeader {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px;
+  height: 56px;
```

We should use `pxToRem` for pixel values that are larger than 2px. The helper needs to be imported at the top of the file.

```suggestion
  padding: pxToRem(16px);
  height: pxToRem(56px);
```